html-forms/index.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>HTML Forms</title>
+  </head>
+  <body>
+    <h1>Coding Form!</h1>
+    <form action="" method="get">
+      <div class="text">Email</div>
+      <div>
+        <label for="email"></label>
+        <input type="text" name="email" size="12" />
+      </div>
+      <div>Why is coding cool?</div>
+      <div>
+        <select name="Select an option">
+          <option value="cool">Because it's cool</option>
+          <option value="interesting">Because it's interesting</option>
+          <option value="powerful">Because it's powerful</option>
+        </select>
+      </div>
+      <p>Favorite Language (choose one)</p>
+      <div>
+        <input type="radio" name="language" value="html" />HTML
```

Comment:
each of these input should have a label, you can wrap the label around the input if desired. That will maintain the click functionality